### DISCUSSION TITLE
Google Index Problem

### DISCUSSION THREAD

**Discussion by SritherSD** [PrivateBin/PrivateBin#1230] [Q&A] [answered]
### Describe the problem/question

Hi Iam trying to index homepage of one of my privatebin instance in google

I want to show my website in google like below website
![2](https://github.com/PrivateBin/PrivateBin/assets/155966740/ebef5f8b-10e6-4675-9799-a8ccece1188f)

But google index blocked by robots.txt.I changed my robots.txt correctly even though index blocked by robots.txt 
![1](https://github.com/PrivateBin/PrivateBin/assets/155966740/ed1cb7e7-f455-40e7-9e4a-09e2a5041632)

anyone can help me to solve this issue?

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1.
2.
...

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by nain-F49FF806** [comment, marked as answer]
@SritherSD You need to remove or customize this file on your server https://github.com/PrivateBin/PrivateBin/blob/master/robots.txt
